api/all.yaml

```diff
@@ -1490,6 +1832,206 @@ components:
+          type: integer
+          format: int64
+        updated_at:
+          description: Time at which this model version was last modified, in epoch milliseconds.
+          type: integer
+          format: int64
+        model_version_id:
+          description: Unique identifier for the model version.
+          type: string
+    ModelVersionStatus:
+      description: |
+        All possible model version statuses (MODEL_VERSION_STATUS_UNKNOWN, PENDING_REGISTRATION, FAILED_REGISTRATION, READY).
+        PENDING_REGISTRATION status indicates that the client has not completely written all model artifacts to external storage.
+        FAILED_REGISTRATION status indicates that the client has failed to write all model artifacts to external storage.
+        READY status indicates that the client has successfully written all model artifacts to external storage.
+      type: string
+      enum:
+        - MODEL_VERSION_STATUS_UNKNOWN
+        - PENDING_REGISTRATION
+        - FAILED_REGISTRATION
+        - READY
+    CreateRegisteredModel:
+      type: object
+      properties:
+        model_name:
+          description: Name of model, relative to parent schema.
+          type: string
+        catalog_name:
+          description: Name of parent catalog.
+          type: string
+        schema_name:
+          description: Name of parent schema relative to its parent catalog.
+          type: string
+        storage_location:
+          description: Storage root URL for registered model (for **MANAGED**, **EXTERNAL** regisetered models)
+          type: string
+        comment:
+          description: User-provided free-form text description.
+          type: string
+      required:
+        - model_name
+        - catalog_name
+        - schema_name
+    UpdateRegisteredModel:
+      type: object
+      properties:
+        comment:
+          description: User-provided free-form text description.
+          type: string
+        new_name:
+          description: New name for the model.
+          type: string
+    ListRegisteredModelsResponse:
+      type: object
+      properties:
+        model_versions:
+          description: An array of model information objects.
+          type: array
+          items:
+            "$ref": "#/components/schemas/RegisteredModelInfo"
+        next_page_token:
+          type: string
+          description: |
+            Opaque token to retrieve the next page of results. Absent if there are no more pages.
+            __page_token__ should be set to this value for the next request (for the next page of results).
+    CreateModelVersion:
+      type: object
+      properties:
+        model_name:
+          description: Name of model, relative to parent schema.
+          type: string
+        catalog_name:
+          description: Name of parent catalog.
+          type: string
+        schema_name:
+          description: Name of parent schema relative to its parent catalog.
+          type: string
```

Comment:
nit: shouldn't the ordering be model-> schema -> catalog?

Reply:
The existing documentation in databricks lists them in this order.  I think we should keep that ordering.
https://docs.databricks.com/api/workspace/registeredmodels/create#name

And the documentation's naming is incorrect for model version.....its not name, its model_name.  I'll fix when I update the databricks documentation to include the missing APIs.